Automattic/wp-calypso
# Add/reader tags newline 3

#### Proposed Changes

* Moves reader tags to a new line to allow for new styling

#### Testing Instructions

* Create a post (that you follow on reader) with more than 3 tags and see how this looks on reader. It should match styling below;

![Image](https://user-images.githubusercontent.com/5634774/184981937-5bc27b4d-b297-49ce-ac4c-2e42d314037b.png)

* Check that the tags show up as follows on the Reader feeds page i.e. http://calypso.localhost:3000/read/feeds/108002187
Related to https://github.com/Automattic/wp-calypso/issues/66622
<img width="492" alt="Screenshot 2022-08-29 at 20 26 43" src="https://user-images.githubusercontent.com/5560595/187282238-6cd70b45-06b5-4b83-953c-ba3a8a003377.png">
* There should be a maximum of 3 tags shown by default and when you click the extra `+` button, you should see the remaining tags
<img width="467" alt="Screenshot 2022-08-29 at 20 28 01" src="https://user-images.githubusercontent.com/5560595/187282458-7dabe39f-cf3b-42be-87a6-86f856bf4fdf.png">


*  Check that the tags show in the reader feed post page i.e. http://calypso.localhost:3000/read/feeds/108002187/posts/4215957581
<img width="473" alt="Screenshot 2022-08-29 at 20 28 58" src="https://user-images.githubusercontent.com/5560595/187282613-94f4c766-f56b-46d6-bf09-8ae61046dfad.png">
* This show a maximum of 5 tags by default and a `+` extra button if more than 5 tags are on the post.
